index.html

```diff
@@ -46,13 +52,34 @@ <h1 class="logo__text">Renome</h1>
                                     <a href="#" class="header__link">menu</a>
                                 </li>
                                 <li>
-                                    <a href="#" class="header__link">reservation</a>
+                                    <a href="#" class="header__link">reservations</a>
                                 </li>
                                 <li>
                                     <a href="#" class="header__link">blog</a>
                                 </li>
                                 <li>
-                                    <a href="" class="header__link">features</a>
+                                    <a class="header__link" id="submenu">features</a>
+                                        <button class="header__linkbutton" id="button_submenu">
+                                            <div class="arrow_right"></div>
+                                        </button>
+                                    <ul class="header__subnav" id="sub_nav">
+                                        <li>
+                                            <a>
+                                                <button class="header__sublinkbutton" id="button_back">
```

Comment:
why header__sublinkbutton and not a header__sublink-button?

Reply:
changed